Extensions/CLIExtensionTemplate.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We should add schema identifier. It could be used for clearly distinguishing between "Extensions catalog item" json file and "Extensions description" json file (and in the future it could also be used for validation of the file).

Reply:
Good catch, I forgot to update those